jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2212,13 +2212,15 @@
 								"{{HealthCheckGroup}}",
 								"apps",
 								"conf",
-								"{{HealthCheckWar}}",
-								"HEALTH-CHECK-4.0"
+								"healthcheck-webapp-1.0.3.war",
```

Comment:
This also looks like a CTP health check dependency. Can we replace this test to run a postman deployed JVM?